graphql_compiler/compiler/cypher_helpers.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+"""Helper functions for Cypher, such as generating unique vertex names within fold scopes."""
+from graphql_compiler.compiler.helpers import FoldScopeLocation, Location
```

Comment:
In library packages, prefer relative imports instead: `from .helpers import <stuff>`. Python import rules are complex and relative imports tend to break less often in libraries.